### ISSUE TITLE
Questions about training configuration for VOC

### ISSUE THREAD

**Issue by jioluoyihao** [cwq159/PyTorch-Spiking-YOLOv3#37]
Hello, I train VOC07+12 and get 48 mAP, far from your 55.50...
Does the training configuration need to be changed, such as learning rate
Thank you very much.

**Comment by cwq159** [comment]
train on VOC07+12
test on VOC2007
no need to be changed about the training config

**Comment by LASTLINEK** [comment]
Could you please tell me about how to construct the dataset and code to train and test on VOC? @jioluoyihao

**Comment by Bender-L** [comment]
> 
https://github.com/Dignity-ghost/PyTorch-Spiking-YOLOv3.git
